lib/membrane/video_compositor/implementations/wgpu/wgpu.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Membrane.VideoCompositor.Wgpu do
+  @moduledoc """
+  This module implements video composition in wgpu
+  """
+  @behaviour Membrane.VideoCompositor.FrameCompositor
+
+  alias Membrane.VideoCompositor.Implementations.OpenGL.Native.Rust.Position
+  alias Membrane.VideoCompositor.Implementations.OpenGL.Native.Rust.RawVideo
```

Comment:
1. You can alias those in one line, using `Membrane.VideoCompositor.Implementations.OpenGL.Native.Rust.{Position, RawVideo}`. 
2. If we are using the same structs for OpenGL and WGPU, we shouldn't name the module `Membrane.VideoCompositor.Implementations.OpenGL.Native.Rust`. Such a name suggests, that this struct is only used in OpenGL implementation. We can separate those structs definitions to other modules like: `Membrane.VideoCompositor.Native.Rust.Position` and `Membrane.VideoCompositor.Native.Rust.RawVideo`.